lib/recommendations/getRecommendations.ts

```diff
@@ -0,0 +1,22 @@
+import { doc, getDoc } from "firebase/firestore";
+
+import getSpotifyData from "@/lib/getSpotifyData";
+
+import { profilesCol } from "../firestore";
+
+const getRecommendations = async (userId: string) => {
+  const profile = await getDoc(doc(profilesCol, userId));
+
+  const artist1 = profile.data()?.topArtists[0].id ?? "4NHQUGzhtTLFvgF5SZesLK";
```

Comment:
```suggestion
  const data = profile.data();
  const artist1 = data?.topArtists[0].id ?? "4NHQUGzhtTLFvgF5SZesLK";
```

Reply:
<img width="1513" alt="image" src="https://user-images.githubusercontent.com/7117993/227438825-cb216e34-760a-4fbe-bb44-6c181510affb.png">
This crashes when no topArtists exist

Reply:
<img width="1513" alt="Screenshot 2023-03-24 at 02 10 04" src="https://user-images.githubusercontent.com/7117993/227438947-0cc89ae1-b2f1-48ec-908a-3af00995b194.png">
or when the topArtists array is empty

Reply:
This is likely an error with how the FirestoreProfile is typed, but might be with this code